lib/model-interfaces/langchain/functions/request-handler/adapters/shared/meta/llama2_chat.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from langchain.schema import AIMessage, HumanMessage
+from langchain.memory import ConversationBufferMemory
+from langchain.prompts import PromptTemplate
+
+
+Llama2ChatPrompt = """[INST] <<SYS>>
+You are an helpful assistant that provides concise answers to user questions with as little sentences as possible and at maximum 3 sentences. You do not repeat yourself. You avoid bulleted list or emojis.
+<</SYS>>
+
+{chat_history}
+
+{input} [/INST]"""
```

Comment:
When using this prompt with the SM endpoint, the resulting prompt sent to the LLM becomes 
```
<s> [INST] [INST] <<SYS>>
You are an helpful assistant that provides concise answers to user questions with as little sentences as possible and at maximum 3 sentences. You do not repeat yourself. You avoid bulleted list or emojis.
<</SYS>>
{chat_history}
{input} [/INST] [/INST]
```

I would recommend that you remove the initial and final `[INST]` tags from the prompt and add them instead in the `transform_input` function. Alternatively, leave the promps as is and modify the `transform_input` function to remove them.